lib/handler.js

```diff
@@ -50,9 +49,12 @@ function writeErrorToResponse(err, req, res, options) {
 
   options = options || {};
 
-  if (res._header) {
```

Comment:
we're removing the case for `res._header` because it no longer exists and replaced by `headersSent`?

Reply:
`headersSent` is the documented api while `_header` is an internal property.